src/server/database.js

```diff
@@ -19,6 +19,28 @@ export const clearDB = () => {
   });
 };
 
+export const putPromise = (key, value) => {
+  return new Promise((resolve, reject) => {
```

Comment:
Good! we really needed these.
but how bout just 'put' or 'get'?
Cause we dont named with 'promise' if those functions return Promise object.